lib/avromatic/model/types/boolean_type.rb

```diff
@@ -8,11 +8,15 @@ def value_classes
           VALUE_CLASSES
         end
 
+        def name
+          'boolean'
```

Comment:
Should these constant name values be frozen?

Reply:
I'll freeze them but I'll also put up a separate PR to add `# frozen_string_literal: true` pragmas to this project too.